app/containers/Applications/ApplicationCommentsByForm.tsx

```diff
@@ -46,6 +73,9 @@ export const ApplicationCommentsByForm: React.FunctionComponent<Props> = props =
               type="checkbox"
               onChange={resolveComment}
             />
+            <Button size="sm" onClick={deleteComment}>
+              Delete
+            </Button>
           </td>
         ) : null}
       </tr>
```

Comment:
We are displaying createdAt below the comment here. We should also show the name of the user who created the column (like Hamza's XD). There is a ciipUserByCreatedBy foreign key connection in reviewComment that can be used to get the user's name.

Reply:
Added user name below the comment